cleanlab/internal/validation.py

```diff
@@ -123,7 +131,7 @@ def assert_indexing_works(X, idx=None, length_X=None):
         raise TypeError(msg)
 
 
-def labels_to_array(y):
+def labels_to_array(y: np.ndarray) -> Any:
```

Comment:
the possible valid types for `y` would include:
list, np.ndarray, np.generic, pd.Series, pd.DataFrame